src/openms/include/OpenMS/KERNEL/MSChromatogram.h

```diff
@@ -51,30 +51,26 @@ namespace OpenMS
     @brief The representation of a chromatogram.
     @ingroup Kernel
   */
-  template <typename PeakT>
-  class MSChromatogram :
-    private std::vector<PeakT>,
+
+  class OPENMS_DLLAPI MSChromatogram :
```

Comment:
I wonder whether some of the small utility functions should not stay in the header for performance reasons? Or does LTO take care of that?